pyiron_atomistics/vasp/base.py

```diff
@@ -1867,6 +1899,7 @@ def write(self, structure, modified_elements, directory=None):
             structure,
             filename=posixpath.join(directory, "POSCAR"),
             write_species=not do_not_write_species,
+            allow_reordering=not self.allow_structure_reordering
```

Comment:
This is bizarre. I also cannot see why a `not` would be necessary here...... maybe test again?

For that matter, it would be excellent to get this new functionality covered in the test suite. If you haven't written unit tests before, it is probably most efficient to just run down another dev at the institute and sit side by side to write this one.